src/tapis-hooks/streams/measurements/useList.ts

```diff
@@ -0,0 +1,21 @@
+import { useQuery } from 'react-query';
+import { list } from 'tapis-api/streams/measurements';
+import { Streams } from '@tapis/tapis-typescript';
+import { useTapisConfig } from 'tapis-hooks';
+import QueryKeys from './queryKeys';
+
+const useList = (params: Streams.ListMeasurementsRequest) => {
```

Comment:
What is the size of the data returned by ListMeasurements? If the number of measurements is likely to be quite large and exceed a single page of data then perhaps it should use the `UseInfiniteQuery` hook. You can see an example in files listings:

https://github.com/tapis-project/tapis-ui/blob/main/src/tapis-hooks/files/useList.ts

Reply:
This returns a single Measurements object which contains all of the variable, time, and value information. The only limitation would be the size of this single object, but this is a limitation of the Measurements implementation.